beniget/beniget.py

```diff
@@ -146,14 +147,22 @@ class DefUseChains(ast.NodeVisitor):
     >>> alias_def = duc.chains[module.body[0].names[0]]
     >>> print(alias_def)
     c -> (c -> (Call -> ()))
-    '''
+    """
 
     def __init__(self):
         self.chains = {}
         self.locals = defaultdict(list)
 
         # deep copy of builtins, to remain reentrant
-        self._builtins = deepcopy(Builtins)
+        # this line
```

Comment:
Without this, `ipython -c "from beniget import DefUseChains; DefUseChains()"` raises

```
TypeError: can't pickle sqlite3.Connection objects
```